src/streams/StreamScreen.js

```diff
@@ -8,6 +10,12 @@ import { getStreams, getSubscriptions } from '../selectors';
 import { NULL_STREAM, NULL_SUBSCRIPTION } from '../nullObjects';
 import StreamCard from './StreamCard';
 
+const styles = StyleSheet.create({
+  margin: {
```

Comment:
We already do have a style called `smallMarginTop` in `theme.js` that you access through the `styles` property in `this.context`.

You can use this, or create a new one called `doubleMarginTop` in the same file and reuse there.